### PR TITLE
Remove unnecessary `this.get()` calls

### PR DESCRIPTION
**Pull request by Turbo87**
These should not be necessary anymore AFAICT

Resolves https://github.com/rust-lang/crates.io/issues/1795

r? @locks

### CODE REVIEW

File: app/controllers/crate/version.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default Controller.extend({
   keywords: alias('crate.keywords'),
   categories: alias('crate.categories'),
   isOwner: computed('crate.owner_user', 'session.currentUser.id', function () {
-    return this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
+    return this.crate.owner_user.findBy('id', this.session.currentUser?.id);
```
Comment (locks) [inline app/controllers/crate/version.js:29]: none of these are async relationships?

Comment (Turbo87) [reply app/controllers/crate/version.js:29]: hmm, good question, some of them actually are 🤔

**Comment by locks** [comment]
@Turbo87 you have some conflicts here

**Comment by Turbo87** [comment]
yeah, I'm aware. closing this for now :)

**Comment by Turbo87** [comment]
@locks I've rebased this and checked the references to async relationships. Those that are relevant are covered by tests and are still working fine after this change.

**Comment by locks** [comment]
Looking sharp!

@bors r+

**Pull request merged**